service/bookturks/user_views/user_quizarena_view.py

```diff
@@ -62,9 +62,9 @@ def user_quizarena_solve_view(request, quiz_id):
         USER: request.user,
         ALERT_MESSAGE: alert_message,
         ALERT_TYPE: alert_type,
-        'quiz_form': content.get('quiz_form')
+        'quiz_form': content.quiz_form
     }
-    request.session['quiz'] = quiz
+    request.session['quiz'] = content.quiz_model
```

Comment:
Rename content to something which makes more sense like quiz_complete_model ? Or something else if that is repetitive.